cloud-provider-snapshots/mongodb-atlas-cloudprovidersnapshots.json

```diff
@@ -0,0 +1,115 @@
+{
+    "typeName": "MongoDB::Atlas::CloudProviderSnapshots",
+    "description": "This resource allows to take one on-demand snapshot, get all or one cloud provider snapshot and delete one cloud provider snapshot.",
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
+    "definitions": {
+        "apiKeyDefinition": {
+            "type": "object",
+            "properties": {
+                "PublicKey": {
+                    "type": "string"
+                },
+                "PrivateKey": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        }
+    },
+    "properties": {
+        "ProjectId": {
+            "description": "The unique identifier of the project for the Atlas cluster.",
+            "type": "string"
+        },
+        "ClusterName": {
+            "description": "The name of the Atlas cluster that contains the snapshots you want to retrieve.",
+            "type": "string"
+        },
+        "Id": {
+            "description": "Unique identifier of the snapshot.",
+            "type": "string"
+        },
+        "Description": {
+            "description": "Description of the on-demand snapshot.",
+            "type": "string"
+        },
+        "RetentionInDays": {
+            "description": "The number of days that Atlas should retain the on-demand snapshot. ",
+            "type": "integer"
+        },
+        "CreatedAt": {
+            "description": "UTC ISO 8601, formatted point in time when Atlas took the snapshot.",
+            "type": "string"
+        },
+        "ExpiresAt": {
```

Comment:
In order to maintain immutability and maintainability, I think it's better to remove this attribute